report/main.qmd

```diff
@@ -0,0 +1,103 @@
+
+Some observations additionally include the sales of 100% juice, orange juice, and sports drinks, which are recorded as integers. However, these counts are only consistently recorded at Site A, and rarely recorded for Site B and C. There are nine days which are missing counts for the bottled zero-calorie and sugared beverages. Each of these nine observations contains missing data for every sales-related column. Of these nine observations, seven occur in the final week of the study at Site B, and two occur mid-study at Site C.
+
+# Exploratory Analysis
+
+Exploratory data analysis can visualize relationships among variables, and informally give some insight on what results can be expected. Based on the data and statistical questions, we recommend the following figures:
+
+1.  Side-by-side boxplots of zero-calorie or sugary drink sales by variables such as day of the week or site can help identify if there are differences in sales between different levels of the categorical variables.
+
+2.  Plots of the proportion of sugary and zero-calorie sales coming from zero-calore drinks over counts, such as Figure 1, can be used to observe the pattern over time. These plots can also be stratified by other variables, such as intervention to identify differences between groups.
+
+3.  A scatterplot of zero-calorie vs sugary drink sales be used to investigate what relationship they have and potentially identify if they are highly correlated.
+
+Additionally, summary statistics such as means can be calculated for different variables, such as average zero-calorie sales for each day of the week to give a numeric summary.
+
+```{r,echo=FALSE,message=FALSE,warning=FALSE}
+#| label: fig-myFirstFigure
+#| fig-cap: Proportion of total sugary and zero-calorie drink sales coming from zero-calorie drinks over time
+#| 
+library(here)
+library(tidyverse)
+# Read in data
+sugary_bev <- read.csv(file = here("./rawdata/june1data.csv"),
+colClasses = c("DofW" = "factor", "Site" = "factor", "Intervention" = "factor"))
+# Plot percentage of sugary or zero sugar sales accounted for by zero sugar drinks over the counts by site
+sugary_bev %>%
+mutate(percent_zero = ZeroCal/(ZeroCal+Sugary)) %>%
+ggplot(aes(x=Count,y=percent_zero,col=Site))+
+geom_point()+
+geom_smooth(se=F)+
+scale_fill_continuous(guide = guide_legend()) +
+theme(legend.position="bottom")+
+labs(x = "Day of the Experiment",
+     y = "Proportion of Zero-Calorie \nand Sugary Drink Sales")
```

Comment:
I don't understand this axis label. How can it be a proportion of both simultaneously?